website/docs/advanced/targeting/targeting.md

```diff
@@ -147,151 +155,57 @@ The following comparators assume that _Comparison attribute_ and _Comparison val
 
-| #   | Comparison attribute | Comparator | Comparison value                           | Served value |
-| --- | -------------------- | ---------- | ------------------------------------------ | ------------ |
-| 1   | Email                | CONTAINS   | `susan@mycompany.com, simon@mycompany.com` | OFF          |
-| 2   | Email                | CONTAINS   | `@mycompany.com`                           | ON           |
-
-All other cases: OFF
-
-### To all other users, serve
-
-This value will be served as a fallback if none of the above rules apply or a [`User Object`](advanced/user-object.md) is not passed to the [ConfigCat SDK](sdk-reference/overview.md) correctly within your application.
-
-## Targeting a percentage of users
-
-With percentage-based user targeting, you can specify a randomly selected fraction of your users whom a feature will be enabled or a different value will be served.
-
-### Enable feature
+:::tip
+The ConfigCat SDKs don't provide date and time values. You have to pass them to the SDKs as custom attributes in UNIX timestamp format.
+:::
 
-1. <a href="https://app.configcat.com/auth/login" target="_blank">Log in</a> to access the _Dashboard_
-2. Go to **Feature Flags & Settings**
-3. Select **TARGET % OF USERS** after clicking the actions icon.
+| Comparator | Description                                                  |
+| ---------- | ------------------------------------------------------------ |
+| BEFORE     | True if _Comparison attribute_ is before _Comparison value_. |
+| AFTER      | True if _Comparison attribute_ is after _Comparison value_.  |
 
-<img src="/docs/assets/targeting-2.png" className="zoomable" alt="targeting-2" />
+##### Example
+If you want to target users who registered before a certain date, you have to pass the registration date to the SDK as a custom attribute in UNIX timestamp format.
 
-## Anatomy of the percentage-based targeting
+```js
+```cs
+var user = new User(appUserData.Identifier)
+{
+    Custom =
+    {
+        ["RegisteredAt"] = User.AttributeValueFrom(appUserData.RegistrationTime),
+    }
+};
+```
 
-Percentage based targeting consists of **% value** and the **Served value** pairs.
+#### Array comparators
 
-### % value
+The following comparators assume that _Comparison attribute_ and _Comparison value_ contain arrays.
 
-Any _number between 0 and 100_ that represents a randomly allocated fraction of your users.
+| Comparator                | Description                                                                              |
+| ------------------------- | ---------------------------------------------------------------------------------------- |
+| ARRAY CONTAINS ANY OF     | True if _Comparison attribute_ contains any of the values in _Comparison value_.         |
+| ARRAY NOT CONTAINS ANY OF | True if _Comparison attribute_ does not contain any of the values in _Comparison value_. |
 
```

Comment:
Ennek is lett amúgy segédfüggvénye, mint a User.AttributeValueFrom(appUserData.RegistrationTime), a dátum típusnál, hogy könnyebben be lehessen adni. Lehet érdemes megmutatni azt is.

Valami ilyesmi, de @adams85 tud szerintem fixen jó példát adni:
```
var user = new User(appUserData.Identifier)
{
    Custom =
    {
        ["UserGroups"] = User.AttributeValueFrom(new []{"Admin", "SuperAdmin}),
    }
};
```

Reply:
Így van, és ez a példa teljesen jó. (C#-ban a `params` paraméter miatt igazából a tömblétrehozást se kell kiírni, de ez így talán jobban mutatja, hogy itt tömbről/listáról van szó).

Egyébként a number comparatarokhoz is van még ilyen helper: 

```cs
var user = new User(appUserData.Identifier)
{
    Custom =
    {
        ["Rating"] = User.AttributeValueFrom(4.5),
    }
};
```